test/Conversion/TppToLoops/tpp-to-loops.mlir

```diff
@@ -1,149 +1,202 @@
-  // CHECK:     memref.store %[[max]], %arg0[%[[i]], %[[j]]] : memref<3x3xf32>
-  // CHECK:   }
-  // CHECK: }
-  tpp.relu out(%arg0: memref<3x3xf32>)
+  // CHECK:     memref.store %[[max]], %[[ARG0]][%[[i]], %[[j]]] : memref<3x3xf32>
+  tpp.relu ins(%arg0: memref<3x3xf32>) out(%arg0: memref<3x3xf32>)
   return
 }
 
 // -----
 
-func.func @add_to_loops(%arg0: memref<3x3xf32>, %arg1: memref<3x3xf32>) {
+// CHECK: func.func @relu_to_loops(
+// CHECK-SAME:  %[[ARG0:.+]]: memref<3x3xf32>, %[[ARG1:.+]]: memref<3x3xf32>)
+func.func @relu_to_loops(%arg0: memref<3x3xf32>, %arg1: memref<3x3xf32>) {
   // CHECK-DAG: %[[ub:.*]] = arith.constant 3 : index
   // CHECK-DAG: %[[lb:.*]] = arith.constant 0 : index
   // CHECK-DAG: %[[step:.*]] = arith.constant 1 : index
+  // CHECK-DAG: %[[relu:.*]] = arith.constant 0.000000e+00 : f32
   // CHECK: scf.for %[[i:.*]] = %[[lb]] to %[[ub]] step %[[step]] {
   // CHECK:   scf.for %[[j:.*]] = %[[lb]] to %[[ub]] step %[[step]] {
-  // CHECK:     %[[load1:.*]] = memref.load %arg0[%[[i]], %[[j]]] : memref<3x3xf32>
-  // CHECK:     %[[load2:.*]] = memref.load %arg1[%[[i]], %[[j]]] : memref<3x3xf32>
-  // CHECK:     %[[add:.*]] = arith.addf %[[load1]], %[[load2]] : f32
-  // CHECK:     memref.store %[[add]], %arg1[%[[i]], %[[j]]] : memref<3x3xf32>
-  // CHECK:   }
-  // CHECK: }
-  tpp.add ins(%arg0: memref<3x3xf32>) out(%arg1: memref<3x3xf32>)
-  return
+  // CHECK:     %[[load:.*]] = memref.load %[[ARG0]][%[[i]], %[[j]]] : memref<3x3xf32>
+  // CHECK:     %[[max:.*]] = arith.maxf %[[load]], %[[relu]] : f32
+  // CHECK:     memref.store %[[max]], %[[ARG1]][%[[i]], %[[j]]] : memref<3x3xf32>
+  tpp.relu ins(%arg0: memref<3x3xf32>) out(%arg1: memref<3x3xf32>)
+  return 
 }
 
 // -----
 
-func.func @identity_to_loops(%arg0: memref<3x3xf32>, %arg1: memref<3xf32>) {
+// CHECK: func.func @add_to_loops(
+// CHECK-SAME:  %[[ARG0:.+]]: memref<3x3xf32>,
+// CHECK-SAME:  %[[ARG1:.+]]: memref<3x3xf32>) {
+func.func @add_to_loops(%arg0: memref<3x3xf32>, %arg1: memref<3x3xf32>) {
   // CHECK-DAG: %[[ub:.*]] = arith.constant 3 : index
   // CHECK-DAG: %[[lb:.*]] = arith.constant 0 : index
   // CHECK-DAG: %[[step:.*]] = arith.constant 1 : index
   // CHECK: scf.for %[[i:.*]] = %[[lb]] to %[[ub]] step %[[step]] {
   // CHECK:   scf.for %[[j:.*]] = %[[lb]] to %[[ub]] step %[[step]] {
-  // CHECK:     %[[tostore:.*]] = memref.load %arg1[%[[j]]] : memref<3xf32>
-  // CHECK:     memref.store %[[tostore]], %arg0[%[[i]], %[[j]]] : memref<3x3xf32>
-  // CHECK:   }
-  // CHECK: }
+  // CHECK:     %[[load1:.*]] = memref.load %[[ARG0]][%[[i]], %[[j]]] : memref<3x3xf32>
+  // CHECK:     %[[load2:.*]] = memref.load %[[ARG1]][%[[i]], %[[j]]] : memref<3x3xf32>
+  // CHECK:     %[[add:.*]] = arith.addf %[[load1]], %[[load2]] : f32
+  // CHECK:     memref.store %[[add]], %[[ARG1]][%[[i]], %[[j]]] : memref<3x3xf32>
+  tpp.add ins(%arg0: memref<3x3xf32>, %arg1: memref<3x3xf32>) out(%arg1: memref<3x3xf32>)
+  return
+}
+
```

Comment:
Can you also add a test for `add_to_loops` with out-of-place `add`? Looks like it is missing.